nautobot/extras/management/commands/runjob.py

```diff
@@ -53,6 +55,11 @@ def handle(self, *args, **options):
             request.id = uuid.uuid4()
             request.user = user
 
+        try:
+            data = json.loads(options.get("data", "{}"))
+        except json.decoder.JSONDecodeError as error:
+            raise CommandError(f"Invalid JSON data:\n{str(error)}")
```

Comment:
Looks like this is failing in UT after the most recent change because `data` is always present in `options`, just defaulting to `None` which isn't loadable JSON. Might need to revert b5af50eeb66d5205a141b6751d89017d867b4c54?

Reply:
removed that commit